openstack/cerebro/cerebro-main/templates/dc-state-service.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dc-state-service
+  labels:
+    app: dc-state-service
+  annotations:
+    prometheus.io/scrape: "true"
+    prometheus.io/port: "{{ .Values.dcStateService.prometheusClientPort }}"
+spec:
+  type: NodePort

```

Comment:
What's the reason for using a `NodePort` over `ClusterIP` here?

Reply:
Changed `NodePort` to `ClusterIP` for the `dc-state-service` and the `optimization-service`.

The `migration-recommender-service` should be reachable from the Nanny. 

Can the Nanny reach the `migration-recommender-service` if we change `NodePort` to `ClusterIP` there?